src/CSSTransitionGroupChild.js

```diff
@@ -157,7 +157,9 @@ class CSSTransitionGroupChild extends React.Component {
   }
 
   render() {
-    return React.Children.only(this.props.children);
+    const props = { ...this.props };
+    Object.keys(propTypes).forEach(key => delete props[key]);
```

Comment:
this pattern feels a bit odd to see here – i'm afraid of people using this with the babel optimization to strip prop types and hitting breakage